akka-bbb-apps/src/main/scala/org/bigbluebutton/SystemConfiguration.scala

```diff
@@ -40,7 +40,7 @@ trait SystemConfiguration extends RedisConfiguration {
   lazy val applyPermissionCheck = Try(config.getBoolean("apps.checkPermissions")).getOrElse(false)
 
   lazy val voiceConfRecordPath = Try(config.getString("voiceConf.recordPath")).getOrElse("/var/freeswitch/meetings")
-  lazy val recordingChapterBreakLenghtInMinutes = Try(config.getInt("recording.chapterBreakLengthInMinutes")).getOrElse(180)
+  lazy val recordingChapterBreakLenghtInMinutes = Try(config.getInt("recording.chapterBreakLengthInMinutes")).getOrElse(0)

```

Comment:
While you're in here, could you fix the typo in the variable name? It says "Lenght", but should be "Length".

Reply:
Done.